frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationItem.tsx

```diff
@@ -212,12 +215,28 @@ export const UtilizationItem: React.FC<UtilizationItemProps> = React.memo(
           </div>
           {!error && (humanAvailable || humanMax) && (
             <div className="co-utilization-card__item-section">
-              <span className="co-utilization-card__item-text">
-                {humanAvailable && <span>{humanAvailable} available</span>}
+              <span
+                className="co-utilization-card__item-text"
+                data-test="utilization-card-item-text"
+              >
+                {humanAvailable && (
+                  <span>
+                    {t('utilization-card~{{humanAvailable}} available', {
+                      humanAvailable,
+                    })}
+                  </span>
+                )}
               </span>
-              <span className="co-utilization-card__item-text">
-                {humanLimit && <span>{humanLimit} total limit</span>}
-                {!humanLimit && humanMax && <span>of {humanMax}</span>}
+              <span
+                className="co-utilization-card__item-text"
+                data-test="utilization-card-item-text"
+              >
+                {humanLimit && (
+                  <span>{t('utilization-card~{{humanLimit}} total limit', { humanLimit })}</span>
+                )}
+                {!humanLimit && humanMax && (
+                  <span>{t('utilization-card~of {{humanMax}}', { humanMax })}</span>
+                )}
```

Comment:
I'm not sure we can break this phrase up like this. You'll want separate properties for the case where there is a just a limit and another property with the full phrase for limit of max.